cpp/serve/request_state.h

```diff
@@ -52,6 +52,10 @@ class RequestModelStateNode : public Object {
   std::vector<SampleResult> committed_tokens;
   /*! \brief The list of input data yet for the model to prefill. */
   Array<Data> inputs;
+  /*! \brief The list of prefilled input data, used to notify prefix cache. */
+  Array<Data> prefilled_inputs;
```

Comment:
How about we do `size_t input_prefill_offset`, which indicates until which pt in inputs we have already prefilled the inputs.

Reply:
`Array<Data> prefilled_inputs_to_notify_prefix_cache`